### DISCUSSION TITLE
Value prediction/regression instead of segmentation

### DISCUSSION THREAD

**Discussion by LRpz** [qubvel/segmentation_models.pytorch#713] [General]
Hi, 

This question might be naive but I think it is still worth asking - could this library be also used for image to image translation tasks - such as denoising? 

If yes, what would be the necessary modifications to apply? I believe it would be on the activation layers and the choice of loss function (MSE) but I do not know more that that.

Thanks!
